service-base/src/main/java/org/eclipse/hono/service/metric/MicrometerBasedMetrics.java

```diff
@@ -69,33 +69,33 @@ public class MicrometerBasedMetrics implements Metrics, SendMessageSampler.Facto
      */
     public static final String METER_CONNECTIONS_ATTEMPTS = "hono.connections.attempts";
     /**
-     * The name of the meter for recording message payload size.
+     * The name of the meter for recording the telemetry/event message payload size.
      */
-    public static final String METER_MESSAGES_PAYLOAD = "hono.messages.payload";
+    public static final String METER_TELEMETRY_PAYLOAD = "hono.telemetry.payload";
     /**
-     * The name of the meter for messages received from devices.
+     * The name of the meter for tracking the processing duration of telemetry/event messages received from devices.
      */
-    public static final String METER_MESSAGES_RECEIVED = "hono.messages.received";
+    public static final String METER_TELEMETRY_PROCESSING_DURATION = "hono.telemetry.processing.duration";
     /**
-     * The name of the meter for recording command payload size.
+     * The name of the meter for recording the command payload size.
      */
-    public static final String METER_COMMANDS_PAYLOAD = "hono.commands.payload";
+    public static final String METER_COMMAND_PAYLOAD = "hono.command.payload";
     /**
-     * The name of the meter for command messages.
+     * The name of the meter for tracking the processing duration of command messages.
      */
-    public static final String METER_COMMANDS_RECEIVED = "hono.commands.received";
+    public static final String METER_COMMAND_PROCESSING_DURATION = "hono.command.processing.duration";
     /**
-     * The name of the meter for queue full events.
+     * The name of the meter for queue-full events.
      */
-    public static final String METER_DOWNSTREAM_FULL = "hono.downstream.full";
+    public static final String METER_AMQP_FULL = "hono.amqp.full";
```

Comment:
can we rename this to *hono.amqp.nocredit*? As there is no notion of a queue in AMQP 1.0 this would better reflect the technical nature of the problem IMHO.